screens/Reader.js

```diff
@@ -1,13 +1,11 @@
 import React from "react";
-import {
-  Animated,
+import {  Animated,
   Dimensions,
   StyleSheet,
   View,
   Platform,
   Text,
-  StatusBar,
-} from "react-native";
+  StatusBar, } from "react-native"; 
```

Comment:
I don't think we need this indentation here. Can this be reset?

Reply:
Fixed